tensorboard/webapp/notification_center/_notification_center_module.ts

```diff
@@ -14,20 +14,40 @@ limitations under the License.
 ==============================================================================*/
 import {NgModule} from '@angular/core';
 import {EffectsModule} from '@ngrx/effects';
-import {StoreModule} from '@ngrx/store';
+import {createSelector, StoreModule} from '@ngrx/store';
 
+import {
+  PersistableSettings,
+  PersistentSettingsConfigModule,
+} from '../persistent_settings';
 import {NotificationCenterDataSourceModule} from './_data_source';
 import {NotificationCenterEffects} from './_redux/notification_center_effects';
 import {reducers} from './_redux/notification_center_reducers';
-import {NOTIFICATION_FEATURE_KEY} from './_redux/notification_center_types';
+import {getLastReadTime} from './_redux/notification_center_selectors';
+import {
+  NOTIFICATION_FEATURE_KEY,
+  State,
+} from './_redux/notification_center_types';
 import {NotificationCenterViewModule} from './_views/views_module';
 
+/** @typehack */ import * as _typeHackStore from '@ngrx/store';
+
+export function getNotificationLastReadTimeSettingSelector() {
+  return createSelector(getLastReadTime, (lastReadTime) => {
+    return {notificationLastReadTimeInMs: lastReadTime};
+  });
+}
+
 @NgModule({
   imports: [
     StoreModule.forFeature(NOTIFICATION_FEATURE_KEY, reducers),
     EffectsModule.forFeature([NotificationCenterEffects]),
     NotificationCenterDataSourceModule,
     NotificationCenterViewModule,
+    PersistentSettingsConfigModule.defineGlobalSetting<
+      State,
```

Comment:
internal cl build fails, please consider fixing it before merging

Reply:
The build was stale and it passes with the latest change. Thanks.